war/src/main/less/base/typography.less

```diff
@@ -0,0 +1,88 @@
+@import '../abstracts/font.less';
+@import '../abstracts/colors.less';
+
+body,
+p {
+  font-family: @font-family-sans;
+  font-size: @font-size-sm;
+  line-height: @line-height-base;
+  color: @text-color;
+}
+
+@media screen and (min-width: 1600px) {
```

Comment:
Why are these repeated and empty?

Reply:
I guess they were a placeholder for responsive typography, that I didn't do in the end. I'll remove them.